examples/todo/packages/contracts/src/codegen/world/IIncrementSystem.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+/* Autogenerated file. Do not edit manually. */
+
+interface IIncrementSystem {
+  function increment() external returns (uint32);
+}
```

Comment:
there's a bunch of leftover autogen in here, think we might need a `pnpm clean` script